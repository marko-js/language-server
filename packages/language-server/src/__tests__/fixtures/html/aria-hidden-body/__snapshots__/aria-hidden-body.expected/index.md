## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <body aria-hidden="true"></body>
    |  ^^^^ Fix any of the following:
  aria-hidden=true should not be present on the document body
```

