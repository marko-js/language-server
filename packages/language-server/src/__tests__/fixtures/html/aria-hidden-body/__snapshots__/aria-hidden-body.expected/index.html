<body data-marko-node-id="0" aria-hidden="true"></body>